sys/linux/fs_ioctl.txt

```diff
@@ -48,6 +51,12 @@ type fscrypt_policy_mode_t[CONTENTS, FILENAMES] {
 
 fs_policy_flags = FS_POLICY_FLAGS_PAD_4, FS_POLICY_FLAGS_PAD_8, FS_POLICY_FLAGS_PAD_16
 
+fsverity_digest {
+	digest_algorithm	int16
+	digest_size		int16
+	digest			buffer[out]
```

Comment:
`buffer` implies a pointer indirection, basically a historical shorthand for `ptr[out, array[int8]]`, see:
https://github.com/google/syzkaller/blob/master/docs/syscall_descriptions_syntax.md
I think we need `array[int8]` here (no pointer indirection).